src/Bridges/graph.jl

```diff
@@ -4,7 +4,7 @@
 # Use of this source code is governed by an MIT-style license that can be found
 # in the LICENSE.md file or at https://opensource.org/licenses/MIT.
 
-const INFINITY = typemax(Int)
+const INFINITY = typemax(Float64)
```

Comment:
Why does the distance/cost need to be a `Float64`? Aren't integer weights sufficient?

Reply:
Because @blegat wanted to use `Inf` to mean that bridging shouldn't happen

Reply:
`ConstraintBridgingCost` and `VariableBridgingCost` are `Float64` anyway. I also thought it might be useful to use fractional sometimes. I can imagine the case when you want to give priority for a bridge but you cannot make the cost below 1 so you're stuck because it is `Int`. With `Float64`, you won't have that kind of artificial limitations. I don't see any advantage of `Int` so the pros-and-cons are easy to compare.